.ci/azure-pipelines/build/windows.yaml

```diff
@@ -32,14 +27,16 @@ steps:
         -DBUILD_tools=OFF ^
         -DBUILD_surface_on_nurbs=ON
     displayName: 'CMake Configuration'
-  - script: cd %BUILD_DIR% && cmake --build . --config %CONFIGURATION%
+  - script: |
+      c: && cd %BUILD_DIR% && cmake --build . --config %CONFIGURATION%
     displayName: 'Build Library'
-  - script: cd %BUILD_DIR% && cmake --build . --target tests --config %CONFIGURATION%
-    displayName: 'Run Unit Tests'
+  - script: |
```

Comment:
`script` has a `workingDir` parameter: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#script

Reply:
We can try it out for sure.

Reply:
Implemented.